### DISCUSSION TITLE
is dbatools and dbachecks safe ?

### DISCUSSION THREAD

**Discussion by tune007** [sqlcollaborative/dbachecks#1025] [Q&A]
I have a very generic question please, have been using dbachecks and dbatools for years but has a friend who is very concerned about the safety of using dbatools and dbachecks and that it could be used for exploitation.  Are there any risks that I should be aware of ?

**Comment by SQLDBAWithABeard** [comment]
I would say to your friend that the supply chain can be trusted, it is safe and locked to only a few people who are all community members. The code is tested and validated  before being released. It is all open source and all code can be seen in public in GitHub

If your friend has any concerns they are free to reach out to one of us for more info.

That said, dbatools allows a person to interact with your instances so it undoubtedly could make life easier for those that wish to do bad thigns as much as those who wish to do good things :-)
